frontend/src/app/shared/confirmation-modal/confirmation-modal.component.html

```diff
@@ -0,0 +1,24 @@
+<lc-base-modal>
```

Comment:
This isn't using the `(dismiss)` output of `lc-base-modal`, so clicking the cross button does not work.